client/src/components/ElementEditor/Header.js

```diff
@@ -192,7 +192,7 @@ class Header extends Component {
               expandable={expandable}
             />
           </div>
-          {!type.broken && <i className={expandCaretClasses} title={expandTitle} />}
+          {!type.broken && <span className={expandCaretClasses} aria-label={expandTitle} title={expandTitle} />}
```

Comment:
Explicit `aria-label` here to match the `title` since some screen readers won't read a `title` attribute.